File: xlsx/php/read/xlsx_read.php
#! /usr/bin/php
<?php
//
//	xlsx_read.php
//
//					Oct/16/2016
//
// ----------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);

include "text_manipulate.php";
include "excel_manipulate.php";
// ----------------------------------------------------------------
fputs (STDERR,"*** 開始 ***\n");

date_default_timezone_set('Asia/Tokyo');

$file_in = $argv[1];
#
print $file_in . "\n";

$dict_aa = excel_read_proc ($file_in);

dict_display_proc ($dict_aa);

echo " Peak memory usage: " . (memory_get_peak_usage(true) / 1024 / 1024) . " MB\r\n";

fputs (STDERR, "*** 終了 ***\n");

// ----------------------------------------------------------------
?>

File: xlsx/csharp/read/xlsx_read.cs
// --------------------------------------------------------
/*
	xlsx_read.cs

				Aug/19/2015


*/
// --------------------------------------------------------
using	System;
using	System.IO;
using	System.Collections.Generic;
using	System.Text;
// --------------------------------------------------------
class	xlsx_read
{
// --------------------------------------------------------
static void Main (string [] args)
{
	Console.WriteLine ("***	開始 ***");

	string	file_xlsx = args[0];
	Console.WriteLine (file_xlsx);

	Dictionary <string,Object> dict_aa
		= xlsx_manipulate.xlsx_to_dict_proc (file_xlsx);

	text_manipulate.dict_display_proc (dict_aa);

	Console.WriteLine ("***	終了 ***");
}

// --------------------------------------------------------

// --------------------------------------------------------
}

// --------------------------------------------------------
